.github/workflows/build_and_test.yml

```diff
@@ -42,25 +42,23 @@ jobs:
     secrets:
       gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
 
-  # Disable due to https://github.com/pytorch/xla/issues/8199
-  # build-torch-with-cuda:
-  #   name: "Build PyTorch with CUDA"
-  #   uses: ./.github/workflows/_build_torch_with_cuda.yml
-  #   needs: get-torch-commit
-  #   with:
-  #     # note that to build a torch wheel with CUDA enabled, we do not need a GPU runner.
-  #     dev-image: us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/development:3.10_cuda_12.1
-  #     torch-commit: ${{needs.get-torch-commit.outputs.torch_commit}}
-  #     runner: linux.24xlarge
+  build-torch-with-cuda:
+    name: "Build PyTorch with CUDA"
+    uses: ./.github/workflows/_build_torch_with_cuda.yml
```

Comment:
I didn't know that we also need to build PyTorch with CUDA. Does building PyTorch with CPU not work for some reason? If we need to build PyTorch with CUDA, do you know if `12.3` is a supported CUDA version to build PyTorch with? From https://github.com/pytorch/pytorch/issues/138609 it looks like upstream picked 12.4 or 12.6

Reply:
Regarding CUDA version, I don't think it's a supported version. However, I believe it should compile fine. That said, I think it would be better to change it to a supported version.

Reply:
Got it. I think we can address this as a follow up.

Reply:
Could you add a TODO here mentioning https://github.com/pytorch/xla/issues/8700